pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -1088,8 +1089,14 @@ public void addTable(@Nonnull TableConfig tableConfig) throws IOException {
         break;
       case REALTIME:
         final String realtimeTableName = tableConfig.getTableName();
```

Comment:
`realtimeTableName` should be the same as the `tableNameWithType`. Let's clean up to use one variable for the same information. You are using `realtimeTableName` for fetching schema while using `tableNameWithType` for exception log.